persian_tools/economic_national_id/exceptions.py

```diff
@@ -0,0 +1,5 @@
+class InvalidEconomicNationalId(Exception):
```

Comment:
Are you using this class anywhere?

Reply:
This class is remained while I was looking for implementing find_place function but there were no information.
I'll get rid of that.